From cd6c81715167333e4de61e7bc1598cbd9cad747d Mon Sep 17 00:00:00 2001
From: Jin Wei <wei.a.jin@intel.com>
Date: Thu, 18 Oct 2012 17:00:48 +0800
Subject: [PORT FROM MAIN] fix valgrind ioctl warning.

BZ: 62797

Initialize local variable to avoid syscall ioctl warning.

Change-Id: I116d047dbd0c8dbe55b5fb1fe3ab3fea988cd907
Orig-Change-Id: I33a90917856018a8527305bb362948ef03bd734c
Signed-off-by: Jin Wei <wei.a.jin@intel.com>
---
 libs/binder/ProcessState.cpp |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/libs/binder/ProcessState.cpp b/libs/binder/ProcessState.cpp
index 294e1d4..6464254 100644
--- a/libs/binder/ProcessState.cpp
+++ b/libs/binder/ProcessState.cpp
@@ -318,7 +318,7 @@ static int open_driver()
     int fd = open("/dev/binder", O_RDWR);
     if (fd >= 0) {
         fcntl(fd, F_SETFD, FD_CLOEXEC);
-        int vers;
+        int vers = 0;
         status_t result = ioctl(fd, BINDER_VERSION, &vers);
         if (result == -1) {
             ALOGE("Binder ioctl to obtain version failed: %s", strerror(errno));
-- 
1.7.4.1

